### PR TITLE
fix(test): add perforce image back to .PHONY

### PR DESCRIPTION
**Pull request by k-dovgan**

### CODE REVIEW

File: tests/docker/perforce/Dockerfile [hunk 1]
```diff
@@ -3,14 +3,14 @@ FROM ubuntu:bionic
...
 
 # Get perforce packages
 RUN wget -q http://package.perforce.com/perforce.pubkey -O - | apt-key add - && \
-    echo "deb http://package.perforce.com/apt/ubuntu bionic release" > /etc/apt/sources.list.d/perforce.list && \
-    apt update
+    echo 'deb http://package.perforce.com/apt/ubuntu bionic release' > /etc/apt/sources.list.d/perforce.sources.list && \
+    apt-get update
 
-RUN apt-get install -y helix-p4d curl
+RUN apt-get install -y curl helix-p4d
```
Comment (i-keliukh) [inline tests/docker/perforce/Dockerfile:13]: This appears to be unrelated change

File: tests/docker/perforce/Dockerfile [hunk 2]
```diff
@@ -3,14 +3,14 @@ FROM ubuntu:bionic
...
 
 # Update package list and install wget
-RUN apt-get update && apt-get install -y wget gnupg2
+RUN apt-get update && apt-get install -y apt-utils wget gnupg2
 
 # Get perforce packages
 RUN wget -q http://package.perforce.com/perforce.pubkey -O - | apt-key add - && \
-    echo "deb http://package.perforce.com/apt/ubuntu bionic release" > /etc/apt/sources.list.d/perforce.list && \
-    apt update
+    echo 'deb http://package.perforce.com/apt/ubuntu bionic release' > /etc/apt/sources.list.d/perforce.sources.list && \
```
Comment (i-keliukh) [inline tests/docker/perforce/Dockerfile:10]: Do the file name or quotes matter?

File: tests/docker/Makefile [hunk 3]
```diff
@@ -5,9 +5,9 @@ ifeq ($(PYTHON),)
 PYTHON := python3.7
 endif
 
-.PHONY: all $(DOCKER_TARGETS) run_perforce
+.PHONY: all $(DOCKER_TARGETS) run_perforce perforce
```
Comment (i-keliukh) [inline tests/docker/Makefile:8]: Please modify the first line instead

Comment (k-dovgan) [reply tests/docker/Makefile:8]: As already explained before, modification of first line will create images 'perfoce-python3.6', 'perforce-python3.7', 'perforce-python3.8' and 'perforce-python3.9', and I can't think of a reason for doing so.

Comment (i-keliukh) [inline tests/docker/Makefile:8]: Please rename DOCKER_TARGETS variable.

**Review by i-keliukh — APPROVED** [review]
